llvm/test/Transforms/InferAddressSpaces/zext-ptrint-conversion.ll

```diff
@@ -0,0 +1,42 @@
+; RUN: opt -S -o - -passes=infer-address-spaces -assume-default-is-flat-addrspace %s | FileCheck %s
+
+target datalayout = "e-p:64:64-p1:64:64-p2:32:32-p3:32:32-p4:64:64-p5:32:32-p6:32:32-p7:160:256:256:32-p8:128:128-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-v2048:2048-n32:64-S32-A5-ni:7:8"
+
+; CHECK-LABEL: @zext_ptrint_conversion(
+; CHECK-NEXT: store i32 0, ptr addrspace(3) %{{.*}}
+; CHECK-NEXT: ret void
```

Comment:
Can you include full generated checks?

Reply:
It is a full check, as follow
define void @zext_ptrint_conversion(ptr addrspace(3) %x) {
  store i32 0, ptr addrspace(3) %x, align 4
  ret void
}